src/components/Form/Form.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { Todo, User } from '../../types';
+
+import { InputTitle } from './InputTitle';
+import { SelectUser } from './SelectUser';
+import { getNewId } from '../../utils/getNewId';
+
+type Props = {
+  users: User[];
+  todos: Todo[];
+  onAddTodo: (todo: Todo) => void;
+};
+
+export const Form: React.FC<Props> = ({ users, todos, onAddTodo }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+
+  const [hasTitleError, setHesTitleError] = useState(false);
+  const [hasUserIdError, setHesUserIdError] = useState(false);
+
+  const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHesTitleError(false);
+  };
+
+  const onUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setHesUserIdError(false);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setHesTitleError(title === '');
+    setHesUserIdError(userId === 0);
```

Comment:
Ensure that the error state variables `setHasTitleError` and `setHasUserIdError` are used consistently with the correct spelling to avoid runtime errors.